packages/localize/test/date/formatDate.test.js

```diff
@@ -232,4 +232,43 @@ describe('formatDate', () => {
     // @ts-ignore tests what happens if you use a wrong type
     expect(formatDate(date)).to.equal('');
   });
+
+  it('displays the appropriate date after post processor', async () => {
+    /**
+     * Post processor method
+     *
+     * @param {string} str
+     * @param {string} [locale='']
+     * @returns {string}
+     */
+    const postProcessor = (str, locale = '') => {
+      const LOCALE_FILTER = ['de-DE', 'en-US'];
+
+      if (LOCALE_FILTER.includes(locale)) {
+        const result = str.split(',');
+        result[0] = result[0].toUpperCase();
+        return result.join();
+      }
+
+      return str;
+    };
+    const testDate = new Date('2012/05/21');
+    const options = {
+      weekday: 'long',
+      year: 'numeric',
+      month: 'long',
+      day: '2-digit',
+      postProcessor,
+    };
+
+    expect(formatDate(testDate, options)).to.equal('Monday, 21 May 2012');
+    localize.locale = 'nl-NL';
+    expect(formatDate(testDate, options)).to.equal('maandag 21 mei 2012');
+    localize.locale = 'fr-FR';
+    expect(formatDate(testDate, options)).to.equal('lundi 21 mai 2012');
+    localize.locale = 'de-DE';
+    expect(formatDate(testDate, options)).to.equal('MONTAG, 21. Mai 2012');
+    localize.locale = 'en-US';
+    expect(formatDate(testDate, options)).to.equal('MONDAY, May 21, 2012');
+  });
```

Comment:
Will there be a test that handles `__datePostProcessors ` as well? (when options.postProcessor is not provided)?

Is it allowed to add multiple post processors? In what order will they be executed and what happens when you provide both a global and a local one?


I know it's still a draft, so maybe you were already planning to implement this :)

Reply:
I added more tests for the global and local processors 😄 

The strategy followed is that we process first global processors in the `options` property and after that, we launch the localize processors. What do you think?

Reply:
Thanks. Looks great to me 👍